packages/bpk-component-nudger/src/BpkNudger.module.scss

```diff
@@ -39,7 +41,7 @@
     }
 
     &--numeric {
-      $size: $bpk-button-line-height + ($bpk-button-padding-y * 2);
+      $size: bpk-spacing-lg() + (bpk-spacing-md() * 2);
```

Comment:
For the `$bpk-button-line-height`, has this changed to aid with the new grid size, as keeping it in use causes the size to be off?

Reply:
`$bpk-button-line-height` uses a spacing token under the hood and also it's used for setting the width